SDK/AppCenterRum/Microsoft.AppCenter.Rum/Microsoft.AppCenter.Rum.csproj

```
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

We should revert and remove `xmlns`. Otherwise, cake script won't work.

```suggestion
<Project Sdk="Microsoft.NET.Sdk">
```

---

Not true, I already verified cake build